src/components/DownloadImageDialog/version.ts

```diff
@@ -0,0 +1,113 @@
+import uniq from 'lodash/uniq';
+import partition from 'lodash/partition';
+import { Dictionary, OsVersion } from './models';
+
+export type VersionSelectionOptions = {
+	title: string;
+	value: string;
+	isRecommended?: boolean;
+	osType: string;
+	line?: string;
+	knownIssueList: string | null;
+} & (
+	| {
+			hasPrebuiltVariants: false;
+			rawVersion: string;
+	  }
+	| {
+			hasPrebuiltVariants: true;
+			rawVersions: {
+				dev?: string;
+				prod?: string;
+			};
+	  }
+);
+
+export const transformVersions = (versions: OsVersion[]) => {
+	// Get a single object per stripped version, with both variants of it included (if they exist). It expects a sorted `
```

Comment:
> It expects a sorted `

Perhaps I am just not understanding, is this a complete thought? 😓

Reply:
nice catch, that was an old comment, not sure where it come from, btw it was probably a sorted `OsVersion array`. I'll remove it as it is not really necessary.